lib/babe/helpers_test.go

```diff
@@ -43,7 +43,7 @@ var (
 		Digest: types.NewDigest(),
 	}
 
-	genesisBABEConfig = &types.BabeConfiguration{
+	westendBABEConfig = &types.BabeConfiguration{
```

Comment:
I agree with the rename but its values should match the westend's real values otherwise this name does not make much sense

Reply:
upon further investigation this isnt even a needed variable since it is only used once, so can just be defined locally. Plus, it is in a helper function that I hope to remove